sljit_src/sljitConfig.h

```diff
@@ -59,6 +59,13 @@ extern "C" {
 /* #define SLJIT_CONFIG_AUTO 1 */
 /* #define SLJIT_CONFIG_UNSUPPORTED 1 */
 
+/* --------------------------------------------------------------------- */
+/*  Compat                                                               */
+/* ----------------------------------------------------------------------*/
+
+/* Define to 1 if your system has secure_getenv function */
+/* #undef HAVE_SECURE_GETENV */
```

Comment:
This file is for sljit configuration, not system cability configuration. This should be done autmatically if needed.

Reply:
this is just a fallback from when sljit is compiled standalone or it is used inside a codebase that doesn't have a facility for system configuration.

when sljit is used as part of another code that has that capability (ex: pcre) then those variables will be set automatically and used accordingly.

tested with both ./configure and cmake, but for the later you will need to use pcre trunk

Reply:
There is a section `Custom defines` above for this purpose.

Reply:
I was considering "custom defines" as something that should be always empty upstream and only to be used for application (meaning the one that is include sljit, not the library itself, which in this case includes the ProtExecAllocator) specific code.

indeed, my intention is to use this section more as you could see in carenas/sljit@041f6b7eb7a1f43f8e1b9bb31e666fdbe756b22b which is a fix for pcre bug 2455